kuttl-test.yaml

```diff
@@ -28,7 +28,6 @@ testDirs:                      # The actual kuttl tests
   - ./tests/kuttl/tests
 crdDir: ./config/crd/bases     # CRDs needed for our operator
 manifestDirs:                  # YAMLs to apply before each test
```

Comment:
Is it okay to have this key without a value?

Reply:
it seems kuttl handles that, but yeah the clean way would be to set it to {}

Reply:
Ah, no worries then if kuttl doesn't complain

Reply:
done